### ISSUE TITLE
Restart Service does'nt work anymore

### ISSUE THREAD

**Issue by ThibautLM** [Cyr-ius/hass-livebox-component#34]
Before it works flawlessly since 3 weeks i can't restart my livebox throught your integration :( 

The error :

`[2658529992] Error handling message: Unknown error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 25, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 525, in handle_execute_script
    await script_obj.async_run(msg.get("variables"), context=context)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1216, in async_run
    await asyncio.shield(run.async_run())
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 350, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 368, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 568, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1491, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1526, in _execute_service
    await handler.job.target(service_call)
  File "/config/custom_components/livebox/__init__.py", line 113, in async_box_restart
    await bridge.async_reboot()
  File "/config/custom_components/livebox/bridge.py", line 168, in async_reboot
    await self.async_make_request(self.api.system.reboot)
  File "/config/custom_components/livebox/bridge.py", line 67, in async_make_request
    return await self._hass.async_add_executor_job(call_api, kwargs)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
TypeError: reboot() takes 1 positional argument but 2 were given`

Thanks in advance

**Comment by Nicolas1er** [comment]
Hello same for me :(

**Comment by lesensei** [comment]
I've tried to look a bit into this with my very limited python knowledge. AFAICT, the [aiosysbus library](https://github.com/Cyr-ius/aiosysbus) used by this component doesn't have a problem rebooting the box, so the problem is indeed in the way this component (or the HA intermediary stuff) calls said library. I'm unfortunately unable to dig deeper into this at the moment, but remain very interested in this (my livebox 2.4GHz WiFi oftentimes stops working altogether, making much of my home automation stuff non-functional, and this service call was what I used to reboot the livebox and make it all work again with an azutomation... now I have to reboot it manually whenever I become aware of the problem).
